src/components/app/routes/loaders/rootLoader.js

```diff
@@ -195,14 +195,21 @@ export const makeRedeemablePoliciesQuery = ({ enterpriseUuid, lmsUserId }) => ({
  * @returns
  */
 const fetchCouponCodes = async (enterpriseUuid) => {
-  const results = await Promise.all([
-    fetchCouponsOverview(enterpriseUuid),
-    fetchCouponCodeAssignments(enterpriseUuid),
-  ]);
-  return {
-    couponsOverview: results[0].data,
-    couponCodeAssignments: results[1].data,
-  };
+  try {
+    const results = await Promise.all([
+      fetchCouponsOverview(enterpriseUuid),
+      fetchCouponCodeAssignments(enterpriseUuid),
+    ]);
+    return {
+      couponsOverview: results[0].data,
+      couponCodeAssignments: results[1].data,
+    };
+  } catch (e) {
+    if (e.response?.status === 404) {
+      return {};
+    }
+  }
+  return null;
```

Comment:
[consideration] Assuming these endpoints do indeed throw 404s during the "happy" path (e.g., no coupons configured for the customer) and we need special handling... if the special, expected case is a 404, would we want to throw the error since it's unexpected?